multiqc/utils/search_patterns.yaml

```diff
@@ -204,6 +204,8 @@ cutadapt:
     contents: 'This is cutadapt'
     # contents: 'cutadapt version' # Use this instead if using very old versions of cutadapt (eg. v1.2)
     shared: true
+damageprofiler:
+    fn: 'dmgprof.json'
```

Comment:
Sometimes nice to add an `*` at the start of a file pattern like this in case people add a filename prefix.